chia/pools/pool_puzzles.py

```diff
@@ -160,10 +160,10 @@ def create_travel_spend(
         delay_ph,
     )
     if is_pool_member_inner_puzzle(inner_puzzle):
-        # inner sol is (spend_type, pool_reward_amount, pool_reward_height, extra_data)
-        inner_sol: Program = Program.to([1, 0, 0, bytes(target)])
+        # inner sol is key_value_list ()
+        inner_sol: Program = Program.to([[("ps", bytes(target))], 0])
```

Comment:
Comment: e.g.  "ps" = Pool State

Reply:
Do we want to change this label to a single byte identifier? My worry is that we are paying for this label.

Reply:
I say use `p`